resources/clj-kondo.exports/com.gfredericks/test.chuck/config.edn

```diff
@@ -1,4 +1,9 @@
-{:hooks
+{:lint-as
+ {com.gfredericks.test.chuck.generators/for clojure.core/for
```

Comment:
I THINK these are correct, but please let me know if not.

Reply:
hmm, depends on what the assumption is

e.g., `for` has this `:parallel` clause, will that mess anything up?

```
(gen'/for [:parallel [n1 gen/nat
                      n2 gen/nat]
           :when ^{:max-tries 20} (coprime? n1 n2)
           :let [product (* n1 n2)]]
  {:n product, :factors [n1 n2]})```

Reply:
Yes, good call. I'll do a better job here and write a proper hook that we can ship with the library. Should get to this soon, thank you for the review!